doc/source/library_reference/common/lib/camera/config.rst

```diff
@@ -0,0 +1,56 @@
+Base Class
+==========
+
+.. currentmodule:: opencsp.common.lib.camera.Camera
+
+.. automodule:: opencsp.common.lib.camera.Camera
+   :members:
+   :special-members: __init__
+   :undoc-members:
+   :show-inheritance:
+   :member-order: groupwise
+
+Image Acquisition Classes
+=========================
+
+.. currentmodule:: opencsp.common.lib.camera.ImageAcquisitionAbstract
+
+.. automodule:: opencsp.common.lib.camera.ImageAcquisitionAbstract
+   :members:
+   :special-members: __init__
+   :undoc-members:
+   :show-inheritance:
+   :member-order: groupwise
+
+.. currentmodule:: opencsp.common.lib.camera.ImageAcquisition_DCAM_color
+
+.. automodule:: opencsp.common.lib.camera.ImageAcquisition_DCAM_color
+   :members:
+   :special-members: __init__
+   :undoc-members:
+   :show-inheritance:
+   :member-order: groupwise
+
+UCamera Classes and Utility Functions
```

Comment:
Should ImageAcquisition_DCAM_mono.py and ImageAcquisition_MSMF.py be included here too?